### DISCUSSION TITLE
Using PiHole as the DHCP server

### DISCUSSION THREAD

**Discussion by Eric-Vignola** [unifi-utilities/unifi-common#386] [Ideas]
[The documentation on running PiHole is excellent](https://github.com/unifi-utilities/unifios-utilities/tree/main/run-pihole). It would be great to add additional steps on how to set PiHole as the UDM's DHCP server.

Main reason for this is so one can monitor which client is being blocked versus a single 10.0.5.1 client.

**Comment by gatesry** [comment]
Have you tried setting the DHCP server option in your UDM network?

**Reply by Eric-Vignola** [reply]
yes, the doc should mention that the UDM's DHCP server needs to be set to `Relay` (pointing to 10.0.5.3) and not `None` (because that would hose the network). And then mention what to fill in the fields under PiHole's DHCP server settings.

my point is simply that these steps should  be added to the doc for completeness. 

sidenote: the `--cap-add=NET_ADMIN \` command needs to be added to `custom_pihole_dote.sh` so PiHole can be used as the DHCP server.

**Reply by gatesry** [reply]
Excellent. I would submit a PR to have the doc updated.
